web/pandas_web.py

```diff
@@ -473,7 +525,13 @@ def main(
                 # Python-Markdown doesn't let us config table attributes by hand
                 body = body.replace("<table>", '<table class="table table-bordered">')
                 content = extend_base_template(body, context["main"]["base_template"])
+
             context["base_url"] = "".join(["../"] * os.path.normpath(fname).count("/"))
+            if selected_language != context["translations"]["default_language"]:
+                context["base_url"] = "".join(
+                    ["../"] * (os.path.normpath(fname).count("/") - 1)
+                )
+
```

Comment:
With the above comment on how to build the target this may not be needed, no?

Reply:
See comment above on navbar_info.